### ISSUE TITLE
Settings issue

### ISSUE THREAD

**Issue by teampros** [thilomaurer/pwcalc#14]
Error: Requiring Clutter, version none: Typelib file for namespace 'Clutter' (any version) not found

Stack trace:
  @/home/me/.local/share/gnome-shell/extensions/pwcalc@thilomaurer.de/prefs.js:32:7
  _getExtensionPrefsModule@resource:///org/gnome/shell/extensionPrefs/main.js:59:13
  _selectExtension@resource:///org/gnome/shell/extensionPrefs/main.js:74:31
  _onCommandLine@resource:///org/gnome/shell/extensionPrefs/main.js:320:17
  main@resource:///org/gnome/shell/extensionPrefs/main.js:635:5
  @<main>:1:43

——————————-
Gnome 3.32
Linux 5.2.14

**Comment by thilomaurer** [comment]
Ok, if you have no `clutter` libs installed, please try installing them:

on Ubuntu try:
```
sudo apt-get install gir1.2-clutter-1.0 gir1.2-clutter-gst-3.0 gir1.2-gtkclutter-1.0
```
on Fedora try:
```
sudo dnf install clutter clutter-gst3 clutter-gtk
```

I hope that works.

I'm searching for a generic solution to this... It turns out I'm only using Clutter to identify keypress events
```js
if (kv==Clutter.Delete||kv==Clutter.BackSpace) removeSelectedItem();
```
Eventually I can use something different for this.

**Comment by thilomaurer** [comment]
@teampros Does the main dropdown from the icon in the gnome menubar work for you, or is the issue only with the Settings window? - Since the extension also imports `Clutter` for the dropdown...

**Comment by teampros** [comment]
The issue appear only for settings on earlier kernel 5.1.2 + gnome 3.32. Kernel 5.2.14 completely brakes extension but can access pwcalc settings from gnome 3.34. ~\_(-,-)_/~

________________________________

**Comment by thilomaurer** [comment]
Which specific OS is this **kernel 5.1.2 + gnome 3.32** ?

**Comment by teampros** [comment]
5.1.2-arch1-1-ARCH #1 SMP PREEMPT Wed May 15 00:09:47 UTC 2019 x86_64 GNU/Linux

Im a Ruby on Rails junior developer. My choice fell on Archlinux.
________________________________

**Comment by thilomaurer** [comment]
Merged #15
